consensus/src/epoch_manager.rs

```diff
@@ -784,6 +785,12 @@ impl EpochManager {
         peer_id: AccountAddress,
         event: VerifiedEvent,
     ) -> anyhow::Result<()> {
+        if let VerifiedEvent::ProposalMsg(proposal) = &event {
+            observe_block(
+                proposal.proposal().timestamp_usecs(),
+                BlockStage::EPOCH_MANAGER_RECEIVED,
```

Comment:
this is not exactly when it's received, it's already after verification (which takes 10+ms for large block I think), we should probably separate them

Reply:
Done.